classes/output/courseformat/content.php

```diff
@@ -271,6 +271,10 @@ public function export_for_template(\renderer_base $output) {
         $addsection = new $this->addsectionclass($format);
         $data->numsections = $addsection->export_for_template($output);
 
+        // Allow next and back navigation between pages.
+        $sectionnav = new \format_multitopic\output\courseformat\content\sectionnavigation($format, $displaysection->section);
```

Comment:
I'd prefer passing the section object with the ID in it and changing the constructor to deal with this, rather than passing just the section number.
```suggestion
        $sectionnav = new \format_multitopic\output\courseformat\content\sectionnavigation($format, $displaysection);
```